backend/app/core/venue_service.py

```
@@ -104,90 +125,100 @@ def search_venues(self, search_params: VenueSearchParams) -> Tuple[List[Enhanced
-                search_params.longitude, 
-                search_params.radius_km
+                search_params.latitude, search_params.longitude, search_params.radius_km
             )
             query = query.filter(distance_query)
-        
+
         # Pricing filters
         if search_params.max_price_per_hour:
             query = query.filter(
                 or_(
                     EnhancedVenue.base_price_per_hour == None,
```

Ruff (0.11.9)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)